vm-wasmi/src/semantic.rs

```diff
@@ -881,14 +883,21 @@ impl Rules for ConstantCostRules {
             NonZeroU32::new(1024).expect("impossible"),
         )
     }
+
+    fn call_per_local_cost(&self) -> u32 {
+        0
+    }
 }
 
 fn instrument_contract(code: &[u8]) -> Vec<u8> {
     let module =
         wasm_instrument::parity_wasm::elements::Module::from_bytes(code).expect("impossible");
-    let instrumented_module =
-        wasm_instrument::gas_metering::inject(module, &ConstantCostRules, "env")
-            .expect("impossible");
+    let instrumented_module = wasm_instrument::gas_metering::inject(
+        module,
+        host_function::Injector::new(Version1x::ENV_MODULE, Version1x::ENV_GAS),
+        &ConstantCostRules,
+    )
+    .expect("impossible");
```

Comment:
```suggestion
    .unwrap();
```
No point in using `expect` if the message doesn’t provide any information. (In fact, I’m of the opinion that `expect` is barely ever useful).

Reply:
it was there, will leave as is. first, it provides. author was thinking that is impossible.
so i can rely in it. while with unwrap author was not thinkin at all. second, lines can be broken or no acces, but panic word will be here. easy to find in code.

Reply:
If all expect calls use `impossible` message that they don’t make it any easier to find the code.  I also disagree that using unwrap means that author thinks the unwrap never happens.